owncloudApp/src/main/java/com/owncloud/android/presentation/ui/settings/fragments/SettingsSecurityFragment.kt

```diff
@@ -155,25 +155,23 @@ class SettingsSecurityFragment : PreferenceFragmentCompat() {
         } else if (prefBiometric != null) {
             biometricManager = BiometricManager.getBiometricManager(activity)
 
-            // Disable biometric lock if Passcode or Pattern locks are disabled
-            if (prefPasscode?.isChecked == false && prefPattern?.isChecked == false) {
-                disableBiometric()
-            }
-            prefBiometric?.setOnPreferenceChangeListener { preference: Preference?, newValue: Any ->
-                val incomingValue = newValue as Boolean
-
-                // Biometric not supported
-                if (incomingValue && biometricManager?.isHardwareDetected == false) {
-                    showMessageInSnackbar(getString(R.string.biometric_not_hardware_detected))
-                    return@setOnPreferenceChangeListener false
+            if (biometricManager?.isHardwareDetected == false) { // Biometric not supported
+                screenSecurity?.removePreference(prefBiometric)
```

Comment:
Reformat this file 👍

Reply:
It is already formatted with Android Studio guidelines! No changes if I click on "Reformat code" 😕

Reply:
My bad :(